Library/ViewModels/DiscoveryViewModel.swift

```diff
@@ -69,9 +69,23 @@ public protocol DiscoveryViewModelType {
   var outputs: DiscoveryViewModelOutputs { get }
 }
 
+private func initialParam() -> DiscoveryParams {
+  let recExperiment = AppEnvironment.current.config?.abExperiments.filter {
+    $0.key == Experiment.Name.defaultToRecs.rawValue
+  }
+
+  if recExperiment?.first?.value == Experiment.Variant.experimental.rawValue {
```

Comment:
Here we will also need to check if the currentUser is nil, because if he/she logs out, we will have to show `All Projects` instead. The if statements will be something like: if AppEnvironment.current.currentUser != nil && recExperiment?.first?.value == Experiment.Variant.experimental.rawValue { ...